huaweicloud/services/ces/resource_huaweicloud_ces_alarmrule.go

```diff
@@ -293,6 +313,7 @@ func getAlarmCondition(d *schema.ResourceData) alarmrule.ConditionOpts {
 		opts.Value = condition["value"].(int)
 		opts.Unit = condition["unit"].(string)
```

Comment:
Some rules have no unit, shall we change the type to pointer.

Reply:
not need, this is a forcenew and omitempty parameter